algoliasearch/index.go

```diff
@@ -445,7 +445,6 @@ func (i *Index) DeleteByQuery(query string, params Params) (res BatchRes, err er
 	copy := duplicateMap(params)
 	copy["attributesToRetrieve"] = []string{"objectID"}
 	copy["hitsPerPage"] = 1000
-	copy["distinct"] = false
```

Comment:
Because distinct is an integer not a boolean, we could put 0 but I don't really see the point either.